clms-angular/src/app/modules/courses/pages/course-modules-page/course-modules-page.component.ts

```
@@ -1,10 +1,188 @@
+        this.httpClient.post('/api/courses/modules/delete-bulk', deletedModules.map(item => item.id)).subscribe(() => {
+          this.fetchModules();
+        });
+      }
+    })
+  }
+
+  onActiveIndeciesChanged($event: number | number[]) {
+    // generate hashmaps for quick lookup
+    const activeModules = Array.isArray($event) ? $event : [$event];
+    const activeModuleMap = {}
+    activeModules.forEach(index => activeModuleMap[index] = true);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)